cmd/registry/registryList.go

```diff
@@ -66,62 +78,33 @@ func NewCmdRegistryList() *cobra.Command {
 				}
 			}
 			existingNodes = client.NodeFilterByRoles(existingNodes, []k3d.Role{k3d.RegistryRole}, []k3d.Role{})
+
 			// print existing registries
-			if len(existingNodes) > 0 {
-				printNodes(existingNodes, headersOff)
+			headers := &[]string{}
+			if !registryListFlags.noHeader {
+				headers = &[]string{"NAME", "PORT", "ROLE", "CLUSTER"} // TODO: add status
```

Comment:
`PORT` is here, but not actually printed further down.

Reply:
:facepalm: